### PR TITLE
LPS-147612 | Validates translation can be synced in selector and text field

### PR DESCRIPTION
**Pull request by Hijky**
**Background**
Create automation tests for Translation Manager

**Test Plan**

Create a page with the widget JS Components Sample
Click AUI Tag translation language menu
Input different language text in AuI Tag field
Select language in AuI Tag selector
Assert translated language is available to select
Assert language selector is changed to the selected language
Assert translation results displayed by the selected language

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-147612

### CODE REVIEW

**Review by zhangyan0701 — CHANGES_REQUESTED** [review]

**Comment by john-co** [comment]
ci:test:frontend:nocompile

File: modules/apps/frontend-js/frontend-js-test/src/testFunctional/paths/TranslationManager.path [hunk 1]
```diff
@@ -12,12 +12,12 @@
...
 	<td>SELECT_TRANSLATION_LANGUAGE_MENU</td>
-	<td>//div[h3='${key_title}']//button[contains(@title,'select-translation-language')]//*[name()='svg'][contains(@class,'lexicon-icon-${key_currentLocale}')]</td>
+	<td>//div[h3='${key_title}']//button[@*='select-translation-language']//*[name()='svg'][contains(@class,'lexicon-icon-${key_currentLocale}')]</td>
 	<td></td>
 </tr>
 <tr>
-	<td>SELECT_TRANSLATION_LANGUAGE_AUI_TAG_ADMINMENU</td>
-	<td>//button[contains(@id,'aui_2d_2Menu')]//*[name()='svg'][contains(@class,'lexicon-icon-${key_currentLocale}')]</td>
+	<td>SELECT_TRANSLATION_LANGUAGE_AUI_TAG_MENU</td>
+	<td>//div[h3='${key_title}']//button[contains(@id,'aui_2d_${key_number}Menu')]//*[name()='svg'][contains(@class,'lexicon-icon-${key_currentLocale}')]</td>
```
[the remark this answers is not in the document]
Comment (Hijky) [reply modules/apps/frontend-js/frontend-js-test/src/testFunctional/paths/TranslationManager.path]: Thanks, I edit it.

**Review by zhangyan0701 — APPROVED** [review]

**Comment by john-co** [comment]
Looks good! :sunglasses:

**Comment by john-co** [comment]
ci:forward

**Pull request closed without merging**